pkg/apis/v2/notificationmanager_types.go

```diff
@@ -61,6 +61,8 @@ type NotificationManagerSpec struct {
 	DefaultConfigSelector *metav1.LabelSelector `json:"defaultConfigSelector,omitempty"`
 	// Receivers to send notifications to
 	Receivers *ReceiversSpec `json:"receivers"`
+	// The default namespace which the secret be in.
```

Comment:
The default namespace which the secret be in.
=>
The default namespace to which notification manager secrets belong